cmd/notation/plugin/install.go

```diff
@@ -136,14 +136,13 @@ func install(command *cobra.Command, opts *pluginInstallOpts) error {
 		if pluginURL.Scheme != "https" {
 			return fmt.Errorf("failed to download plugin from URL: only the HTTPS scheme is supported, but got %s", pluginURL.Scheme)
 		}
-		tmpFile, err := os.CreateTemp("", notationPluginDownloadTmpFile)
+		tmpDir, err := os.MkdirTemp("", pluginDownloadTmpDir)
 		if err != nil {
-			return fmt.Errorf("failed to create temporary file required for downloading plugin: %w", err)
+			return fmt.Errorf("failed to create temporary directory: %w", err)
```

Comment:
Is it better to provide the temp directory path in the error so the users can check whether they have proper permission or storage size for the directory?

Reply:
Resolving as discussed offline.